package.json

```diff
@@ -55,8 +55,13 @@
 		"build:clean": "rm -rf node_modules && rm -f package-lock.json && npm i && npm run build"
 	},
 	"engines": {
-		"node": ">=20.18.0",
-		"npm": ">=10.8.2"
+		"node": ">=20.18.0"
+	},
+	"devEngines": {
```

Comment:
devEngines: non-standard props, which tool recognizes it ?

Reply:
thought it was a proposal only, now I see https://github.com/npm/cli/pull/7766 is merged last October. thanks